style.css

```diff
@@ -348,10 +356,401 @@ footer {
+    list-style: none;
+    color: #3a4a42;
+    font-size: 12px;
+    font-weight: 600;
+    justify-content: center;
+    padding: 10px;
+    align-items: center;
+    top: 89px;
+    left: 20px;
+    height: 32px;
+    margin-top: 2px;
+    margin-bottom: 20px;
+    gap: 8px;
+  }
+
+  .stories li {
+    border-radius: 3px;
+    background-color: #ebf0ee;
+    padding: 5px;
+  }
+
+  .headline-section {
+    height: 556px;
+    width: 720px;
+  }
+
+  .works {
+    display: none;
+  }
+
+  .work {
+    color: #172b4d;
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 52px;
+    margin-top: 50px;
+  }
+
+  .indicator {
+    display: none;
+  }
+
+  .other-section {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    flex-direction: column;
+    grid-gap: 10px;
+    justify-content: center;
+    align-items: center;
+    margin-top: 80px;
+  }
+
+  p {
+    color: #172b4d;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 32px;
+    text-align: center;
+    margin-bottom: 10px;
+    margin-top: 0;
+    width: 610px;
+  }
+
+  .last-part {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 5px 5px;
+    width: 700px;
+    height: 490px;
+  }
+
+  .remarks > p {
+    font-size: 20px;
+    line-height: 30px;
+    width: 363px;
+    justify-content: center;
+    align-items: center;
+    margin-top: 200px;
+    flex-grow: 0;
+    margin-left: 150px;
+  }
+
+  .footer-objects {
+    display: flex;
+    position: absolute;
+  }
+
+  .shape3 {
+    display: flex;
+    position: absolute;
+    right: -192px;
+    top: -200px;
+  }
+
+  .shape4 {
+    display: flex;
+    position: absolute;
+    left: 300px;
+    bottom: -50px;
+  }
+
+  .shape5 {
+    display: flex;
+    position: absolute;
+    left: 1010px;
+    bottom: -180px;
+  }
+
+  .footer-form > p {
+    display: none;
+  }
+
+  .footer-form {
+    width: 50%;
+  }
+
+  .nav-menu {
+    display: none;
+  }
+
+  .labels {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: auto;
+    list-style-type: none;
+    gap: 10px;
+    margin-bottom: 50px;
+    margin-top: -15px;
+  }
+
+  .indicator2 {
+    display: none;
+  }
+
+  .get-button1 {
+    font-size: 17px;
+    font-weight: 400;
+    line-height: 12px;
+    color: #fff;
+    background-color: #36b37e;
+    text-align: center;
+    margin: 0;
+    padding: 20px;
+    border-radius: 4px;
+    display: flex;
+    width: 140px;
+    justify-content: center;
+    border: none;
+  }
+
+  .get-button1:hover {
+    box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
+  }
+
+  .get-button1:active {
+    box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
+    background: #008552;
+  }
+
+  .social-media {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 1rem;
+    list-style: none;
+    margin-top: -60;
+    padding: 0;
```

Comment:
[OPTIONAL]
- [ ] For your `margin-top`, kindly note that you must put a unit after your number.